sdg/components.py

```diff
@@ -47,9 +48,10 @@ def sdg_op(
     model = getenv("model")
     endpoint = getenv("endpoint")
 
-    if sdg_ca_cert := getenv("SDG_CA_CERT_PATH"):
+    if use_tls:
```

Comment:
Do we need to parametrize this? Instead can we do something like 
```suggestion
    if getenv("SDG_CA_CERT_PATH"):
```
This way we just default to TLS if there is a cert and not if there is not one? Would there be a case where the user would have a cert in their configmap, but would not want it to be used?

Reply:
So SDG_CA_CERT_PATH will always point to a truthy value (/tmp/cert/ca.crt), so this conditional will always trigger.  Wed have to actially check the path for existance of the file which is a bit more involved

Reply:
Is it more involved then: 
```
sdg_ca_cert = getenv("SDG_CA_CERT_PATH")
if os.path.exists(sdg_ca_cert):
```

Or do we need to actually read the file and validate its contents too?

Reply:
i think we can do something like this, probably should ensure the file is actually populated / non-empty too, but that's an easy check